Packs/DSPM/Integrations/DSPM/DSPM.yml

```diff
@@ -0,0 +1,984 @@
+      description: 'Asset field type.'
+      type: String
+    - contextPath: DSPM.AssetFields.dataTypes.name
+      description: 'Asset field datatype name.'
+      type: String
+    - contextPath: DSPM.AssetFields.dataTypes.label
+      description: 'Asset field datatype label.'
+      type: String
+    - contextPath: DSPM.AssetFields.dataTypes.hitPercentage
+      description: 'Asset field datatype hitPercentage.'
+      type: Number
+    - contextPath: DSPM.AssetFields.dataTypes.maskedValues.masked_value
+      description: 'Asset field datatype masked value.'
+      type: String
+    - contextPath: DSPM.AssetFields.dataTypes.maskedValues.line
+      description: 'Asset field datatype masked value line.'
+      type: Number
+    - contextPath: DSPM.AssetFields.schemaName
+      description: 'Asset field schemaName.'
+      type: String
+  - arguments: []
+    name: dspm-get-data-types
+    description: 'Fetches the available data types for the DSPM integration.'
+    outputs:
+    - contextPath: DSPM.DataTypes.Key
+      description: 'Data types key.'
+      type: String
+    - contextPath: DSPM.DataTypes.No
+      description: 'Data types number.'
+      type: Number
+  - arguments: []
+    description: Returns list of label names based on company
+    name: dspm-list-labels
+    outputs:
+    - contextPath: DSPM.Label.Key
+      description: 'Label key.'
+      type: String
+    - contextPath: DSPM.Label.No
+      description: 'Label number.'
+  - arguments:
+    - description: List of regions (comma-separated values).
+      isArray: false
+      name: region_in
+      required: false
+      secret: false
+    - description: Exact region.
+      isArray: false
+      name: region_equal
+      required: false
+      secret: false
+    - description: 'List of cloud providers ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"].'
+      isArray: false
+      name: cloud_provider_in
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'AWS'
+      - 'AZURE'
+      - 'GCP'
+      - 'SNOWFLAKE'
+      - 'FILE_SHARE'
+      - 'O365'
+    - description: 'Exact cloud provider ["AWS", "AZURE", "GCP", "SNOWFLAKE",
+      "FILE_SHARE", "O365"].'
+      isArray: false
+      name: cloud_provider_equal
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'AWS'
+      - 'AZURE'
+      - 'GCP'
+      - 'SNOWFLAKE'
+      - 'FILE_SHARE'
+      - 'O365'
+    - description: List of Service Types (comma separated values).
+      isArray: false
+      name: service_type_in
+      required: false
+      secret: false
+    - description: Exact Service Type.
+      isArray: false
+      name: service_type_equal
+      required: false
+      secret: false
+    - description: List of Life Cycles (comma separated values).
+      isArray: false
+      name: lifecycle_in
+      required: false
+      secret: false
+    - description: 'List of project IDs.'
```

Comment:
Is this a comma-separated list?  If so, it should be written as:
A comma-separated list of project IDs.